components/sup/src/util/users.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[cfg(any(target_os="linux", target_os="macos"))]
+extern crate users;
+
+use error::{Result, Error};
+use hcore::package::PackageInstall;
+
+static LOGKEY: &'static str = "USERS";
```

Comment:
Pretty sure we just make them up as we go along. This could be a useful question for @ryankeairns as to whether this output format is even helpful to people